websocket/connection.go

```diff
@@ -210,17 +220,22 @@ func (c *ManagedConnection) Send(msg interface{}) error {
 		return err
 	}
 
-	return conn.WriteMessage(websocket.BinaryMessage, b.Bytes())
+	return c.connection.WriteMessage(websocket.BinaryMessage, b.Bytes())
```

Comment:
why don't we permit user to specify binary or text here?

Reply:
because until now we didn't needed. This was specifically built to encapsulate reconnecting logic for the queue-proxy -> autoscaler websocket connection. I'd say we add this if we need it?

Reply:
Do we expect to receive either, though?